src/main/kotlin/com/team4099/robot2021/commands/shooter/ShootCommand.kt

```diff
@@ -0,0 +1,46 @@
+package com.team4099.robot2021.commands.shooter
+
+import com.team4099.lib.logging.Logger
+import com.team4099.lib.units.derived.rotations
+import com.team4099.lib.units.perMinute
+import com.team4099.robot2021.config.Constants
+import com.team4099.robot2021.subsystems.Feeder
+import com.team4099.robot2021.subsystems.Shooter
+import com.team4099.robot2021.subsystems.Vision
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class ShootCommand : CommandBase() {
+  init{
+    addRequirements(Shooter)
+  }
+
+  override fun initialize(){
+    Shooter.targetVelocity = when(Vision.currentDistance) {
+      Vision.DistanceState.LINE -> 4600.0.rotations.perMinute
```

Comment:
we may want to move this to constants somehow? not sure bc we did it this way last year